### DISCUSSION TITLE
537a4a10-20c4-49c2-bbbd-b32a568e975c

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#26] [General]
# 537a4a10-20c4-49c2-bbbd-b32a568e975c

임베디드가...Swift?! 남일인줄 알았던 WWDC24에서 공개된 Embedded for Swift를 찍먹해 보았습니다.

https://www.varofla.com/537a4a10-20c4-49c2-bbbd-b32a568e975c

**Comment by kwonkun** [comment]
재밌네요 ㅎㅎ

matter 쪽 예제는 포스팅은 아직 안하신거죠??

**Reply by varofla** [reply]
재미있게 봐주셔서 감사합니다 :)
다른 SDK로 matter를 먼저 이해한 후 사용해보려 했는데, 그것조차 못하고 있습니다 ㅜㅜ..
언젠가 여유가 생기면 포스팅해볼게요..!
